test/dynamicImport.test.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env jest */
+import { transform } from 'babel-core'; // eslint-disable-line import/no-extraneous-dependencies
+import plugin from '../src';
+
+// According to https://github.com/tc39/proposal-dynamic-import
+
+describe('import()', () => {
+  const transformerOpts = {
+    babelrc: false,
+    plugins: [
+      // We need to add the corresponding syntax plugin
+      // in order to parse the `import()`-calls
+      'syntax-dynamic-import',
+      [plugin, {
+        root: [
+          './test/testproject/src',
+        ],
+        alias: {
+          test: './test/testproject/test',
+        },
+      }],
+    ],
+  };
+
+  it('CFG should resolve the path based on the root config', () => {
```

Comment:
`CFG` not needed. Usually we start all tests with `should...`. Because `it should...` :)